smartsim/_core/launcher/dragon/dragonLauncher.py

```diff
@@ -368,13 +204,18 @@ def stop(self, step_name: str) -> StepInfo:
         :rtype: StepInfo
         """
 
-        if not self.is_connected:
-            raise LauncherError("Launcher is not connected to Dragon.")
-
         stepmap = self.step_mapping[step_name]
         step_id = str(stepmap.step_id)
+
+        if step_id.startswith("SLURM-"):
```

Comment:
unless this stuff is really just some sort of tweak that we need to do to let the DragonLauncher handle either WLM....

In that case, we should probably try more to avoid having two launchers and just instantiate one.

Reply:
It could, if asked to and both were available. This is just a way to use the schedulers directly and not the launchers interactively.